packages/dcos-launch/extra/launch/config.py

```diff
@@ -49,375 +21,256 @@ def expand_path(path: str, relative_dir: str) -> str:
     return os.path.abspath(os.path.join(relative_dir, path))
```

Comment:
I have a unit test for just this function so it made sense to keep it out. It would probably be a better test if I just instantiated that Validator class and made this private